src/pwncore/routes/team.py

```diff
@@ -11,16 +11,38 @@
 @router.get("/list")
 async def team_list():
     # Do login verification here
+    all_list = await Team.all()
     return [{"team_name": "CID Squad"}, {"team_name": "Astra"}]
 
 
 @router.get("/login")
 async def team_login():
     # Do login verification here
-    return {"status": "logged in!"}
-
-
-@router.get("/members/{team_id}")
-async def team_members(team_id: int):
-    # Get team members from team_id
-    return [{"name": "ABC", "user_id": 3432}, {"name": "DEF", "user_id": 3422}]
+    team = Team(name="test")
+    is_true = await team.check_password("testing") # bool
+    return {"status": is_true}
+
+@router.get("/update/test/{current_points}")
+async def team_change_password(current_points:int):
+    team = Team(name="test")
+    is_true = await team.check_password("testing")
+    update = await team.current_points_fn(update=True,new_value=current_points) #bool
+    return {"status":update}
+
+@router.get("/get/current_points")
+async def get_test():
+    team=Team(name="test")
+    val=await team.current_points_fn(get=True)
+    return {"status":val} # int
+
+@router.get("/members/")
+async def team_members():
+    team=Team(name="test")
+    val=await team.members(get=True) # a dict of members with keys ['member1','member2','member3']
+    return {'status':val}
+
+@router.get("/members/update")
```

Comment:
Try to keep each function's URL endpoint to be of a single depth, for eg. updating members can be just `/update`.

Reply:
Those are just some endpoints which I've written just for the sake of testing the db. @MrigankaDebnath03 is currently working on the endpoints.